drivers/ble/sphero/sphero_r2d2_actions.go

```diff
@@ -0,0 +1,271 @@
+package sphero
+
+type Audio uint16
```

Comment:
opted to put these in a different file being that there are quite a few Audio and Animation options